src/runtime-helpers.js

```diff
@@ -423,74 +424,61 @@ function rewriteActionsWithAdobeAuthAnnotation (packages, deploymentPackages) {
   const REWRITE_ACTION_PREFIX = '__secured_'
 
   // avoid side effects, do not modify input packages
-  const newPackages = { ...packages }
-  const newDeploymentPackages = { ...deploymentPackages }
+  const newPackages = cloneDeep(packages)
```

Comment:
going with clone deep adds a (small) dependency but makes the implementation clearer and less error prone